builder/builder-next/controller.go

```diff
@@ -446,15 +446,16 @@ func getGCPolicy(conf config.BuilderConfig, root string) ([]client.PruneInfo, er
 		} else {
 			gcPolicy = make([]client.PruneInfo, len(conf.GC.Policy))
 			for i, p := range conf.GC.Policy {
-				var b int64
+				var b = defaultKeepStorage
```

Comment:
Or should no default be applied at all here? i.e. we're iterating through gc-policies that are either configured (or set by default?)

Reply:
> (or set by default?)

defaults are set a few lines up, so we wouldn't hit those in this branch; https://github.com/moby/moby/blob/64ca1c0f927af5ebbbeb88a619ee3dbfbd64bdf4/builder/builder-next/controller.go#L444-L448

Reply:
> Or should no default be applied at all here?

Yes, no default at all matches the implementation of buildkit.